mappings/net/minecraft/world/biome/Biome.mapping

```diff
@@ -25,6 +27,11 @@ CLASS net/minecraft/class_1959 net/minecraft/world/biome/Biome
 	METHOD method_21740 getTemperature (Lnet/minecraft/class_2338;)F
 		ARG 1 blockPos
 	METHOD method_24218 calculateSkyColor ()I
+	METHOD method_24374 getNoisePenalty (Lnet/minecraft/class_1959$class_4762;)F
```

Comment:
"noise penalty"?

Reply:
yeah, like penalty between a generator selected noise point and a biome. lowest penalty got picked by the gen

Reply:
yeah but why call it penalty

Reply:
make it something chance related, show it's a relative rarity to the target